api/app/main.py

```diff
@@ -144,16 +147,37 @@ async def get_most_recent_historic_model_values(
         model: ModelEnum, request: schemas.StationCodeList, _: bool = Depends(authenticate)):
     """ Returns the weather values for the last model prediction that was issued
     for the station before actual weather readings became available.
+    NOTE: This api method can be made redundant - calling /models/{model}/predictions/most_recent/
+    will return historic as well as most recent.
     """
     try:
         logger.info('/models/%s/predictions/historic/most_recent/', model.name)
-        historic_predictions = await fetch_most_recent_historic_predictions(model, request.stations)
+        historic_predictions = await fetch_predictions_by_station_code(model, request.stations,
+                                                                       time_utils.get_utc_now())
         return schemas.WeatherModelPredictionResponse(predictions=historic_predictions)
     except Exception as exception:
         logger.critical(exception, exc_info=True)
         raise
 
 
+@api.post('/models/{model}/predictions/most_recent/',
+          response_model=schemas.WeatherStationsModelRunsPredictionsResponse)
+async def get_most_recent_model_values(
```

Comment:
Is this intended to be named this way along with the url? This doesn't say anything about biased adjusted model values

Reply:
Indeed it is. So right now, we have three different calls, that are doing the same damn thing. I want to get rid of the others, and only use this one. This one endpoint, returns the most recent model predictions and bias adjusted values going back 5 days in the past and as far into the future as there is data. So we can get rid of /models/{model}/predictions/historic/most_recent/ and /models/{model}/predictions/

Reply:
Ah I see. Gotcha